src/datasources/transaction-api/transaction-api.module.ts

```diff
@@ -2,7 +2,7 @@ import { Global, Module } from '@nestjs/common';
 import { TransactionApiManager } from './transaction-api.manager';
 import { CacheFirstDataSourceModule } from '../cache/cache.first.data.source.module';
 import { ITransactionApiManager } from '../../domain/interfaces/transaction-api.manager.interface';
-import { ValidationErrorFactory } from '../errors/validation-error-factory';
+import { ValidationErrorFactory } from '../../domain/errors/validation-error-factory';
```

Comment:
Do we need the `ValidationErrorFactory` here?

Reply:
We don't! Thanks for noticing! 🙂

Fixed in [d376ff0](https://github.com/5afe/safe-client-gateway-nest/pull/82/commits/d376ff082174dc6624e1eeb6ec396f4bfbc36045)